reaper/util.py

```diff
@@ -120,7 +120,7 @@ def write_state_file(path, state):
     os.rename(temp_path, path)
 
 
-def create_archive(content, arcname, metadata=None, outdir=None):
+def create_archive(content, arcname, rootdir=True, metadata=None, outdir=None):
```

Comment:
Ah, thanks, fixed. I remember grepping for it, but forgot to follow up on the topic, as I also think there's a [broken usage](https://github.com/scitran/reaper/blob/master/reaper/gephysio.py#L62) in gephysio. (`metadata` and `outdir` is swapped)